R/dimensions.R

```diff
@@ -101,3 +102,19 @@ pitch_international <- list(
   origin_x = 0,
   origin_y = 0
 )
+
+#' @rdname pitch_tracab
+#' @export
+# ChyronHego Tracab: only length & width provided in metadata.
+# Source: official documentation.
+# This solution takes the remaining dimensions from the international pitch.
+make_pitch_tracab <- function(length=105, width=68){
+  pitch_tracab <- lapply(pitch_international, function(x){100.0*x}) # cm
+  pitch_tracab["length"] <- 100.0*length
+  pitch_tracab["width"] <- 100.0*width
+  pitch_tracab["origin_x"] <- -pitch_tracab[["length"]]/2.0
+  pitch_tracab["origin_y"] <- -pitch_tracab[["width"]]/2.0
+  return(pitch_tracab)
```

Comment:
I should set up travis to enforce this but I have been using the [tidyverse style guide](https://style.tidyverse.org/functions.html#return), which advises against `return` in these cases:

> Only use return() for early returns. Otherwise, rely on R to return the result of the last evaluated expression.

Do you have any aversion to sticking with this here?